backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/Interfaces/ISingleRightService.cs

```diff
@@ -30,5 +30,17 @@ public interface ISingleRightService
         /// </param>
         /// <returns></returns>
         Task<HttpResponseMessage> CreateDelegation(string party, DelegationInput delegation);
+
+        /// <summary>
+        ///     Clears cashed acesses of a user
+        /// </summary>
+        /// <param name="party">
+        ///     The party from which the rights have been given (delegator)
+        /// </param>
+        /// <param name="user">
+        ///     The uuid identifier of the user (delegation recipient) to clear access cashe on
+        /// </param>
+        /// <returns></returns>
+        Task<HttpResponseMessage> ClearAccessCasheOnUser(string party, BaseAttribute user);
```

Comment:
My only minor feedback comments (across implementation):
- Typo Cashe -> Cache
- BaseAttribute variable name 'user' would be more covering to use recipient, as this is either a user or a org party

Reply:
Fixed! 😊